src/Razor/src/Microsoft.CodeAnalysis.Remote.Razor/MEFComposition.cs

```diff
@@ -9,16 +9,19 @@
 
 namespace Microsoft.CodeAnalysis.Remote.Razor;
 
-internal abstract partial class RazorServiceFactoryBase<TService>
+internal static class MEFComposition
```

Comment:
Tiny nit: Please don't capitalize acronyms in names unless it falls under the special "two-letter acronym" case.

https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers

Also:

https://sourceroslyn.io/#q=MEF

Reply:
Going to rename this to "RemoteMefComposition" so I don't need to futz with git trying to fix the casing of the file name itself 😁